key.go

```diff
@@ -297,7 +305,7 @@ func NewEventKey(k Key, ch rune, mod ModMask) *EventKey {
 		k = KeyBackspace
 	}
 
-	return &EventKey{t: time.Now(), key: k, ch: ch, mod: mod}
+	return &EventKey{t: time.Now(), key: k, str: str, mod: mod}
```

Comment:
@gdamore This change broke some code a few lines higher up in this function, where you need to assign to str now instead of ch:

<img width="696" height="159" alt="Screenshot 2025-12-19 at 17 36 32" src="https://github.com/user-attachments/assets/aa03651d-41b8-4869-b4fd-4cc84590b8a4" />

I'm finding that using a linter is a really good idea to catch errors like this, highly recommended. This one was found by staticcheck, which is built into gopls.